### PR TITLE
Updates SE-0220’s status to avoid confusion.

### PR DESCRIPTION
**Pull request by jasdev**
(It might be confusing for folks to see this one as implemented, when there was a revert. 😌)

### CODE REVIEW

**Comment by jasdev** [comment]
@airspeedswift, light nudge — any chance we could land this? 🙏🏽 SE-0220 came up again in our team’s chat and it reminded me of this PR hah. 😄

**Comment by benrimmington** [comment]
There's a list of accepted proposals which have either _expired_ or need to be revised:

<https://forums.swift.org/t/addressing-unimplemented-evolution-proposals/40322>

It may be better to keep the status as accepted, and append the date of [acceptance](https://forums.swift.org/t/accepted-se-0220-count-where/15280).

```md
* Status: **Accepted (2018-08-15)**
```

Then the [status page](https://apple.github.io/swift-evolution/) could automatically change how the expired proposals are displayed.

**Comment by jasdev** [comment]
@benrimmington Ah! Great call — added the acceptant date and reverted back the proposal status. ✅

File: proposals/0220-count-where.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 * Proposal: [SE-0220](0220-count-where.md)
 * Author: [Soroush Khanlou](https://github.com/khanlou)
 * Review Manager: [Chris Lattner](https://github.com/lattner)
-* Status: **Accepted**
-* Implementation: [apple/swift#16099](https://github.com/apple/swift/pull/16099)
+* Status: **Accepted (2018-08-15)**
+* Implementation: [apple/swift#16099](https://github.com/apple/swift/pull/16099), [apple/swift#22289](https://github.com/apple/swift/pull/22289) (revert PR)
```
Comment (benrimmington) [inline proposals/0220-count-where.md:8]: Please could you also add the decision notes:

```suggestion
* Decision Notes: [Rationale](https://forums.swift.org/t/accepted-se-0220-count-where/15280), [Additional Commentary](https://forums.swift.org/t/require-parameter-names-when-referencing-to-functions/27048)

```

**Pull request merged**